src/Sulu/Bundle/ContentBundle/Tests/Functional/Controller/NodeResourcelocatorControllerTest.php

```diff
@@ -197,8 +197,13 @@ public function testGenerateWithIncompleteParts()
 
     public function testGetAction()
     {
-        // prepare history nodes
-        $newsData = $this->data[1];
+        // prepare history nodes (get to use the newest hash value)
+        $this->client->request(
```

Comment:
That shouldn't be necessary after https://github.com/sulu/sulu-document-manager/pull/105